src/main/java/io/mosip/mimoto/controller/IssuersController.java

```diff
@@ -78,4 +92,73 @@ public ResponseEntity<Object> getIssuerConfig(@PathVariable("issuer-id") String
         return ResponseEntity.status(HttpStatus.OK).body(responseWrapper);
     }
 
+    @GetMapping("/{issuer-id}/credentials-supported")
+    public ResponseEntity<Object> getCredentialsSupportedForIssuer(@PathVariable("issuer-id") String issuerId,
+                                                                   @RequestParam(required = false) String search) {
+        ResponseWrapper<Object> responseWrapper = new ResponseWrapper<>();
+        responseWrapper.setId(ID);
+        responseWrapper.setVersion("v1");
+        responseWrapper.setResponsetime(DateUtils.getRequestTimeString());
+        IssuerSupportedCredentialsResponse credentialsSupported;
+        try {
+            credentialsSupported = issuersService.getCredentialsSupported(issuerId, search);
+        }catch (ApiNotAccessibleException | IOException exception){
+            logger.error("Exception occurred while fetching issuers ", exception);
```

Comment:
Logging needs to be corrected. Instead of issuers while fetching credentials supported

Reply:
changed